vm/src/stdlib/os.rs

```diff
@@ -2304,31 +2330,6 @@ mod posix {
         unsafe { libc::umask(mask) }
     }
 
-    #[pyattr]
-    #[pyclass(module = "os", name = "uname_result")]
-    #[derive(Debug, PyStructSequence)]
-    struct UnameResult {
-        sysname: String,
-        nodename: String,
-        release: String,
-        version: String,
-        machine: String,
-    }
-
-    #[pyimpl(with(PyStructSequence))]
-    impl UnameResult {}
-
-    #[pyfunction]
-    fn uname(vm: &VirtualMachine) -> PyResult<UnameResult> {
-        let info = uname::uname().map_err(|err| err.into_pyexception(vm))?;
-        Ok(UnameResult {
-            sysname: info.sysname,
-            nodename: info.nodename,
-            release: info.release,
-            version: info.version,
-            machine: info.machine,
-        })
-    }
```

Comment:
You can keep this function here, but just change it to return `super::_os::UnameResult`

Reply:
Do I need to make the `UnameResult` struct public to do that?

Reply:
Oh, yeah, probably `pub(super)`